draft-holmberg-mmusic-t140-usage-data-channel.xml

```diff
@@ -135,42 +131,45 @@
         <t>
            An offerer and answerer MAY, in each offer and answer, include an
            SDP 'dcsa' attribute <xref target="I-D.ietf-mmusic-data-channel-sdpneg"/>
-           in the SDP media descripton (m= line) describing the SCTP association used
+           in the m= section describing the SCTP association used
            to realize the T.140 data channel.
         </t>
-        <t>
-          If included, the 'dcsa' attribute contains the fmtp attribute used to indicate
-          a maximum character transmission rate <xref target="RFC4103"/>. The 'cps'
-          attribute parameter is used indicate the maximum character transmission rate
-          that the endpoint that includes the attribute is able to handle. The 'format' attribute
-          parameter is not used with T.140 data channels, and MUST be set to "-".
-        </t>
-        <t>
-          If not included, it indicates that no maximum character transmission rate is indicated. 
-          It does not mean that the default value of 30 applies <xref target="RFC4103"/>.
-        </t>
-        <t>
-          The offerer and answerer MAY modify the 'cps' attribute parameter value in subsequent
-          offers and answers.
-        </t>
-        <t>
-          NOTE: The 'cps' attribute parameter is especially useful when a T.140 data channel
-          endpoint is acting as a gateway [<xref target="sec.gw"/>] and is interworking with
-          a T.140 transport mechanism that have restrictions on how many characters can be sent
-          per second.
-        </t>
-        <t>
-          Below is an example of the dcsa attribute for an T.140 data channel with a
-          'cps' attribute parameter with a attribute parameter value of 20:
-        </t>
-        <t>
-          a=dcsa:1 fmtp:- cps=20
+        <section anchor="sec.sdp.dcsa.trans" title="Maximum Character Transmission">
+          <t>
+            A 'dcsa' attribute can contain the SDP fmtp attribute used to indicate
+            a maximum character transmission rate <xref target="RFC4103"/>. The 'cps'
+            attribute parameter is used indicate the maximum character transmission rate
+            that the endpoint that includes the attribute is able to handle. The 'format' attribute
```

Comment:
Should it be "receive" instead of "handle"? Because it is reception that can be limited.

Reply:
I will change to "receive".